OWML.ModHelper.Input/ModInputCombination.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Collections.Generic;
+using OWML.Common;
+using UnityEngine;
+
+namespace OWML.ModHelper.Input
+{
+    public class ModInputCombination : IModInputCombination
+    {
+        public float LastPressedMoment => _lastPressedMoment;
+        public float PressDuration => _lastPressedMoment - _firstPressedMoment;
+
+        internal string Combo { get; private set; }
+
+        private bool _isPressed = false, _isFirst = false;
+        private float _firstPressedMoment = 0f, _lastPressedMoment = 0f;
```

Comment:
Booleans default to false, and floats (and all numbers) default to 0, so you don't have to set these values

Reply:
done